packages/mobile/src/invite/saga.ts

```diff
@@ -217,7 +216,7 @@ export function* sendInvite(
 }
 
 function* initiateEscrowTransfer(temporaryAddress: string, e164Number: string, amount: BigNumber) {
-  const escrowTxId = generateStandbyTransactionId(temporaryAddress + '-escrow')
```

Comment:
Do you happen to know why this was needed in the first place? Just want to make sure it's not an identifier for any downstream logic

Reply:
Like @yorhodes said, this was likely to avoid possible conflict with the invite tx but they would need to be created at the same millisecond. Given that web3 has been quietly ripping out the '-escrow' for us and this has never been an issue, it's unlikely this will cause issues.